components/core/Text.tsx

```diff
@@ -1,6 +1,12 @@
 import styled from "@emotion/styled";
 import { Link as ChakraLink, Text } from "@chakra-ui/core";
 import theme from "@/src/theme";
+import Linkify from "react-linkify";
+import React, { ReactElement } from "react";
```

Comment:
Nitpick: the "React" import isn't directly used

Reply:
My IDE was giving me a warning because I was using the `<a></a>` tag 😂